plugin/db/mysql/restore.go

```diff
@@ -93,12 +93,35 @@ func (driver *Driver) replayBinlog(ctx context.Context, originalDatabase, pitrDa
 		return err
 	}
 
+	caseVariable := "lower_case_table_names"
+	identifierCaseSensitive, err := driver.getServerVariable(ctx, caseVariable)
+	if err != nil {
+		return err
+	}
+
+	var originalDBName string
+	switch identifierCaseSensitive {
+	case "0":
```

Comment:
Use enum strings for 0, 1, 2 so that the name of the enum serves as comment for the magic numbers.